test/metabase/task/send_pulses_test.clj

```diff
@@ -138,26 +137,6 @@
                  (pulse-channel-test/send-pulse-triggers pulse-1)
                  (pulse-channel-test/send-pulse-triggers pulse-2))))))))
 
-(deftest init-will-schedule-triggers-test
```

Comment:
Why did you remove this test? As far as I can see we still need it for the new `init-send-pulse-triggers!` function

Reply:
bc it's no longer triggered by calling `(task/init! ::task.send-pulses/SendPulses)` it just happens to run `now,` but it happens on a different thread.

Also this should be covered by the new migration test.

Reply:
> this should be covered by the new migration test.

They're not exactly equivalent, so I'm just checking we're okay weakening the test coverage. This test coverts the property that the pulse trigger's info includes the schedule. The migration test is weaker in that it only tests that there are triggers are created for the pulse.

Reply:
I updated instead of checking for count, we check for trigger info in the migration test: 0f75110a4c